assets/js/util/tracking/index.test.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Internal dependencies
+ */
+import { bootstrapTracking, DATA_LAYER, trackEvent } from './';
```

Comment:
> I wonder: would it make sense to use react-helmet for this instead of our document.append? It's a bit of a nicer API and could make things easier to test, but I don't have strong feelings on this one as we don't have much of a need for it otherwise from what I can tell right now.

The ACs say that the module shouldn't depend on any third-party code.

+1 for this test though. We could mock `document.head.appendChild` but I would rather just check that the element is there 👍

Reply:
Ah, right on. I think this is fine anyway and helmet would be overkill.

Usually with testing-library it renders stuff into a `<div>` and not a full-page… so we may have to test the call to `appendChild`, but I think it's worth exploring this in a separate issue. (#1083 maybe.)